pkg/crc/config/settings.go

```diff
@@ -138,6 +139,9 @@ func RegisterSettings(cfg *Config) {
 	cfg.AddSetting(IngressHTTPSPort, constants.OpenShiftIngressHTTPSPort, validatePort, RequiresHTTPSPortChangeWarning,
 		fmt.Sprintf("HTTPS port to use for OpenShift ingress/routes on the host (1024-65535, default: %d)", constants.OpenShiftIngressHTTPSPort))
 
+	cfg.AddSetting(EnableBundleQuayFallback, false, ValidateBool, SuccessfullyApplied,
+		"Enable pull bundle from quay as fallback (true/false, default: false)")
```

Comment:
Maybe `If bundle download from the default location fails, fallback to quay.io (true/false ...` ?

Reply:
Updated.